ui/components/Flow.tsx

```diff
@@ -21,7 +21,14 @@ export type Values = Partial<
   | UpdateVerificationFlowBody
 >;
 
-export type Methods = "oidc" | "password" | "code" | "totp" | "webauthn";
+export type Methods =
+  | "oidc"
+  | "password"
+  | "code"
+  | "totp"
+  | "webauthn"
+  | "lookup"
+  | "lookup_secret";
```

Comment:
question: is "lookup" an alias for "lookup_secret"?

Reply:
Only `lookup_secret` is valid, I removed the `lookup`.